### PR TITLE
DEV: Update bump2version config

### PR DESCRIPTION
**Pull request by shaneahmed**
- Update bump2version config for citation and docker

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -22,7 +30,7 @@ spellcheck-targets = comments
 dictionaries = en_US,python,technical
 max-cognitive-complexity = 14
 max-expression-complexity = 7
-per-file-ignores = 
+per-file-ignores =
 	test_*: E800
```
Comment (John-P) [inline setup.cfg:34]: Why are we ignoring commented out code in test files?

**Review by John-P — COMMENTED** [review]
Happy with the change to bump2version config but unsure about ignore E800

File: setup.cfg (hunk 1 above)
Comment (shaneahmed) [reply setup.cfg:34]: I think it should flagged on individual lines in the code. Let me create a separate PR for this as I expect there would be errors after removing this.

**Review by John-P — APPROVED** [review]
Looks good. Happy to review flake8 E800 exception in a another PR.

**Pull request merged**